gulpfile.js

```diff
@@ -82,6 +83,66 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  let path = require('path');
+  let rootDir = 'dist';
+
+  let config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      `${rootDir}/1.0/index.html`,
+      `${rootDir}/1.0/shell`,
+      `${rootDir}/1.0/about.html`,
+      `${rootDir}/elements/**`,
+      `${rootDir}/js/*.js`,
+      `${rootDir}/css/*.css`,
+      `${rootDir}/bower_components/**/webcomponents-lite.min.js`,
+      `${rootDir}/bower_components/highlight/highlight.js.js`,
+    ],
+    dynamicUrlToDependencies: {
+      '/1.0/shell.html': [
+        path.join(rootDir, 'templates', 'head-meta.html'),
+        path.join(rootDir, 'templates', 'site-nav.html'),
+      ]
+    },
+    navigateFallback: '/1.0/shell',
+    runtimeCaching: [
+    {
+      urlPattern: /\/images\/*/,
```

Comment:
This picks up images from Google Maps:

![image](https://cloud.githubusercontent.com/assets/1749548/17816231/262e593a-6606-11e6-99c8-e1e531890a44.png)

It looks like it would be a bit hairy to tweak the RegExp to avoid Google Maps images while leaving it flexible enough to pick up any images on this site. So, we can use the *other* way of specifying URL routes, which just uses a string instead of a RegExp, but has the advantage of only matching same-origin URLs by default:

`urlPattern: '/images/(.*)'`

(The `sw-toolbox` [docs](https://googlechrome.github.io/sw-toolbox/docs/master/tutorial-usage.html) explain the different URL routing options, but it's kind of a mess, to be honest. In this particular case, it comes in handy, though.)